docs/manuals/core/concepts/scenario.md

```diff
@@ -5,8 +5,14 @@ As its name implies, with Taipy scenarios, the users can instantiate different v
 problem with different assumptions. This is extremely useful in a business context where impact analysis
 and what-if analysis are essential in the decision process.
 
-After analyzing its first scenario, an end-user may be interested in modifying input
-data nodes (not the intermediate nor the output data nodes), re-running the same pipelines, and
+A scenario contains an executable Directed Acyclic Graph (or DAG). The scenario DAG is a set of tasks connecting
+data nodes together. It can also be broken down into smaller graphs for execution by defining a `Sequence^`s.
+A sequence is a subset of tasks derived from the scenario's set of tasks, forming a smaller executable DAG that
+can be submitted separately from the scenario DAG. A scenario can also contain a set of other data nodes
+outside of the scenario DAG to represent additional data related to the scenario but are not executable.
+
+After analyzing its first scenario, an end-user may be interested in modifying input data nodes
+(not the intermediate nor the output data nodes), re-running the identical sequences or scenario and
```

Comment:
Please refer to the other PR:
https://github.com/Avaiga/taipy-doc/pull/535

"A scenario contains a Directed Acyclic Graph (or DAG) submittable for execution. The scenario DAG is a set of tasks connecting..."

"... A sequence is a subset of connected tasks derived from the scenario's set of tasks, forming a smaller executable DAG that..."

"... disconnected from the scenario DAG to represent additional data related to the scenario. The execution of the scenario does not compute the other data nodes..."

Reply:
have rebased and issue is fixed 😃